include/pybind11/pytypes.h

```diff
@@ -12,6 +12,9 @@
 #include "detail/common.h"
 #include "buffer_info.h"
 
+#include <cstdio>
```

Comment:
is stdio still necessary?

Reply:
Short answer: no, thanks for pointing it out!

Long answer: commit message of 8e145c09f76da8d037e0fd9a25088ce2ca9854aa